browser-ext/background.js

```
@@ -0,0 +1,138 @@
+const extensionAPI = typeof browser !== undefined ? browser : chrome;
```

Biome (2.1.2)

[error] Invalid typeof comparison.

Compare with one of the following string literals:


(lint/correctness/useValidTypeof)